server/proxy/pf_context.c

```diff
@@ -77,7 +77,49 @@ rdpContext* p_client_context_create(rdpSettings* clientSettings,
 	settings->ServerPort = port;
 	settings->SoftwareGdi = FALSE;
 	settings->RedirectClipboard = FALSE;
+
+	/* Client Monitor Data */
+	settings->MonitorCount = clientSettings->MonitorCount;
```

Comment:
maybe wrap in an inline function, this func becomes a mess

Reply:
we need to figure out a way to use freerdp_settings_clone